apps/gportal/src/styles/variables.scss

```diff
@@ -0,0 +1,6 @@
+$gdi-brand-purple: #670e46;
```

Comment:
ah, css variables. Nice. We are still on the fence on how theming will be done in the nuxt/tailwind.

Reply:
I would probably recommend using mixins, and rethink the way variables are passed down to child components. It would also be nice if some of the options could be exposed in the Theme tab. This might be a useful step towards UI-templates.